### PR TITLE
Add `varcheck` linter and fix errors.

### PR DESCRIPTION
**Pull request by Vavilen**
For [#780](https://github.com/skycoin/skycoin/issues/780)

### CODE REVIEW

**Comment by Vavilen** [comment]
@gz-c , I commented unused vars. Please, let me know if I need remove them or leave everything as it was.

File: Makefile [hunk 1]
```diff
@@ -33,8 +37,7 @@ test: ## Run tests
 	go test ./src/... -timeout=1m
 
 lint: ## Run linters. Use make install-linters first.
-	vendorcheck ./...
-	gometalinter --disable-all -E goimports --tests --vendor ./...
+	varcheck ./...
```
Comment (gz-c) [inline Makefile]: why did you remove the other linters? wtf?

File: Makefile [hunk 2]
```diff
@@ -48,7 +49,9 @@ cover: ## Runs tests on ./src/ with HTML code coverage
 install-linters: ## Install linters
 	go get -u github.com/FiloSottile/vendorcheck
 	go get -u github.com/alecthomas/gometalinter
-	gometalinter --vendored-linters --install
+	go get github.com/fzipp/gocyclo
```
Comment (gz-c) [inline Makefile]: dont' change this

File: Makefile [hunk 3]
```diff
@@ -35,6 +35,7 @@ test: ## Run tests
 lint: ## Run linters. Use make install-linters first.
 	vendorcheck ./...
 	gometalinter --disable-all -E goimports --tests --vendor ./...
+	varcheck ./...
```
Comment (gz-c) [inline Makefile]: use `-E varcheck`

File: src/api/webrpc/transaction_test.go [hunk 4]
```diff
@@ -18,9 +18,9 @@ const (
 	txConfirmed = true
 )
 
-var emptyTransactionStr = `{
-        "transaction": null
-    }`
+//var emptyTransactionStr = `{
```
Comment (gz-c) [inline src/api/webrpc/transaction_test.go]: delete

File: src/api/webrpc/webrpc.go [hunk 5]
```diff
@@ -23,8 +23,8 @@ var (
 	errCodeInvalidParams  = -32602 // Invalid params	Invalid method parameter(s).
 	errCodeInternalError  = -32603 // Internal error	Internal JSON-RPC error.
 
-	errMsgParseError     = "Parse error"
-	errMsgInvalidRequest = "Invalid Request"
+	errMsgParseError = "Parse error"
+	//errMsgInvalidRequest = "Invalid Request"
```
Comment (gz-c) [inline src/api/webrpc/webrpc.go]: delete

File: src/cipher/secp256k1-go/secp256k1-go2/ec_test.go [hunk 6]
```diff
@@ -4,54 +4,54 @@ import (
...
-		"3045022100ec6eb6b2aa0580c8e75e8e316a78942c70f46dd175b23b704c0330ab34a86a34022067a73509df89072095a16dbf350cc5f1ca5906404a9275ebed8a4ba219627d6701",
-		"7c8e7c2cb887682ed04dc82c9121e16f6d669ea3d57a2756785c5863d05d2e6a",
-	},
-	{
-		"04ee90bfdd4e07eb1cfe9c6342479ca26c0827f84bfe1ab39e32fc3e94a0fe00e6f7d8cd895704e974978766dd0f9fad3c97b1a0f23684e93b400cc9022b7ae532",
-		"3045022100fe1f6e2c2c2cbc916f9f9d16497df2f66a4834e5582d6da0ee0474731c4a27580220682bad9359cd946dc97bb07ea8fad48a36f9b61186d47c6798ccce7ba20cc22701",
-		"baff983e6dfb1052918f982090aa932f56d9301d1de9a726d2e85d5f6bb75464",
-	},
-}
+//var ta = [][3]string{
```
Comment (gz-c) [inline src/cipher/secp256k1-go/secp256k1-go2/ec_test.go]: leave it commented out like this

File: src/coin/coin_test.go [hunk 7]
```diff
@@ -10,13 +10,13 @@ import (
...
-	genPublic, genSecret        = cipher.GenerateKeyPair()
-	genAddress                  = cipher.AddressFromPubKey(genPublic)
-	testMaxSize                 = 1024 * 1024
-	_genTime             uint64 = 1000
-	_incTime             uint64 = 3600 * 1000
-	_genCoins            uint64 = 1000e6
-	_genCoinHours        uint64 = 1000 * 1000
+	genPublic, genSecret = cipher.GenerateKeyPair()
+	genAddress           = cipher.AddressFromPubKey(genPublic)
+	//testMaxSize                 = 1024 * 1024
```
Comment (gz-c) [inline src/coin/coin_test.go]: delete

File: src/coin/coin_test.go [hunk 8]
```diff
@@ -10,13 +10,13 @@ import (
...
+	_genTime uint64 = 1000
+	//_incTime             uint64 = 3600 * 1000
```
Comment (gz-c) [inline src/coin/coin_test.go]: delete

File: src/daemon/gnet/pool_test.go [hunk 9]
```diff
@@ -16,18 +16,18 @@ import (
 )
 
 const (
-	addr          = "127.0.0.1:50823"
-	addrb         = "127.0.0.1:50824"
-	addrc         = "127.0.0.1:50825"
+	addr = "127.0.0.1:50823"
+	//addrb         = "127.0.0.1:50824"
```
Comment (gz-c) [inline src/daemon/gnet/pool_test.go]: delete these 2

File: src/daemon/gnet/pool_test.go [hunk 10]
```diff
@@ -16,18 +16,18 @@ import (
...
 	port          = 50823
 	address       = "127.0.0.1"
 	silenceLogger = false
 )
 
-var (
-	listener net.Listener
-	conn     net.Conn
-)
+//var (
```
Comment (gz-c) [inline src/daemon/gnet/pool_test.go]: delete

File: src/gui/blockchain.go [hunk 11]
```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/skycoin/skycoin/src/daemon"
 )
 
-const lastBlockNum = 10
+//const lastBlockNum = 10
```
Comment (gz-c) [inline src/gui/blockchain.go]: delete

File: src/util/logging/logging.go [hunk 12]
```diff
@@ -31,14 +31,14 @@ const (
...
 
-var levelNames = []string{
-	"CRITICAL",
-	"ERROR",
-	"WARNING",
-	"NOTICE",
-	"INFO",
-	"DEBUG",
-}
+//var levelNames = []string{
```
Comment (gz-c) [inline src/util/logging/logging.go]: delete

File: src/visor/blockchain_test.go [hunk 13]
```diff
@@ -19,15 +19,17 @@ import (
 var (
 	genPublic, genSecret = cipher.GenerateKeyPair()
 	genAddress           = cipher.AddressFromPubKey(genPublic)
-	testMaxSize          = 1024 * 1024
+	//testMaxSize          = 1024 * 1024
```
Comment (gz-c) [inline src/visor/blockchain_test.go]: delete all of these unused vars in this file

File: src/visor/blockdb/blockchain_test.go [hunk 14]
```diff
@@ -18,11 +18,11 @@ import (
 var (
 	genPublic, genSecret = cipher.GenerateKeyPair()
 	genAddress           = cipher.AddressFromPubKey(genPublic)
-	testMaxSize          = 1024 * 1024
+	//testMaxSize          = 1024 * 1024
```
Comment (gz-c) [inline src/visor/blockdb/blockchain_test.go]: delete all unused

File: src/visor/historydb/historydb_test.go [hunk 15]
```diff
@@ -21,18 +21,19 @@ import (
 var (
 	genPublic, genSecret = cipher.GenerateKeyPair()
 	genAddress           = cipher.AddressFromPubKey(genPublic)
-	testMaxSize          = 1024 * 1024
-	blockBkt             = []byte("blocks")
-	transactionBkt       = []byte("transactions")
-	outputBkt            = []byte("uxouts")
-	addressInBkt         = []byte("address_in")
-	addressOutBkt        = []byte("address_out")
+	//testMaxSize          = 1024 * 1024
```
Comment (gz-c) [inline src/visor/historydb/historydb_test.go]: delete all unused

File: src/visor/readable_test.go [hunk 16]
```diff
@@ -10,14 +10,14 @@ import (
...
 
-const (
-	testMasterKeysFile  = "testmaster.keys"
-	testWalletFile      = "testwallet.wlt"
-	testBlocksigsFile   = "testblockchain.sigs"
-	testBlockchainFile  = "testblockchain.bin"
-	testWalletEntryFile = "testwalletentry.json"
-	testWalletDir       = "./"
-)
+//const (
```
Comment (gz-c) [inline src/visor/readable_test.go]: delete
